paychmgr/manager.go

```diff
@@ -70,6 +73,10 @@ type Manager struct {
 
 func NewManager(ctx context.Context, shutdown func(), sm stmgr.StateManagerAPI, pchstore *Store, api PaychAPI) *Manager {
 	impl := &managerAPIImpl{StateManagerAPI: sm, PaychAPI: api}
+
+	// Add network tag to context
+	ctx, _ = tag.New(ctx, tag.Upsert(metrics.Network, buildconstants.NetworkBundle))
```

Comment:
but where is this context coming from? do we need to track down the source if we know it's not being set here? I'd rather do this closer to the source if we can work it out

Reply:
I'll attach the tag to the context directly here
https://github.com/filecoin-project/lotus/blob/d463f4cdded33df7003215c2d49b6e9e1d6859ea/node/modules/paych.go#L17-L22